### DISCUSSION TITLE
Questions about auto_arima()

### DISCUSSION THREAD

**Discussion by tuttoaposto** [Nixtla/statsforecast#71] [Q&A] [answered]
1. Is it possible to get the bestfit model from the ```auto_arima_f()``` step in ```auto_arima()```? It would be nice to get the same level of details as in ```model.arima_res_.params``` and ```model.summary()``` in ```pmdarima```.
2. Is it possible to enable setting ```max_p```, ```max_q``` , etc in ```auto_arima()```?

Thank you!

**Comment by AzulGarza** [comment, marked as answer]
Hi @tuttoaposto! Let me create this discussion as an issue to work on both features :)

**Reply by tuttoaposto** [reply]
Thanks a lot! I look forward to them.

Another valuable feature to add is .update(). Then we don't need to re-train the models with hourly observations until the models get outdated.
